CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 1.0.2 (next)
 
 * Your contribution here.
+* [#66](https://github.com/rodolfobandeira/spacex/pull/66): Implement API Info endpoint [@fernandomaia](https://github.com/fernandomaia).
```

Comment:
I think the numbers here should be your PR number and not the issue number. So this should be 69  instead of 66 in both the places.